### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by lbittarello** [jmboehm/RegressionTables.jl#3]
The package seems great! I can't install it though. I get the following message:

```
ERROR: RegressionTables's requirements can't be satisfied because of the following fixed packages: DataFrames
resolve(::Dict{String,Base.Pkg.Types.VersionSet}, ::Dict{String,Dict{VersionNumber,Base.Pkg.Types.Available}}, ::Dict{String,Tuple{VersionNumber,Bool}}, ::Dict{String,Base.Pkg.Types.Fixed}, ::Dict{String,VersionNumber}, ::Set{String}) at .\pkg\entry.jl:490
resolve(::Dict{String,Base.Pkg.Types.VersionSet}, ::Dict{String,Dict{VersionNumber,Base.Pkg.Types.Available}}, ::Dict{String,Tuple{VersionNumber,Bool}}, ::Dict{String,Base.Pkg.Types.Fixed}) at .\pkg\entry.jl:479
edit(::Function, ::String, ::Base.Pkg.Types.VersionSet, ::Vararg{Base.Pkg.Types.VersionSet,N} where N) at .\pkg\entry.jl:30
(::Base.Pkg.Entry.##1#3{String,Base.Pkg.Types.VersionSet})() at .\task.jl:335
Stacktrace:
 [1] sync_end() at .\task.jl:287
 [2] macro expansion at .\task.jl:303 [inlined]
 [3] add(::String, ::Base.Pkg.Types.VersionSet) at .\pkg\entry.jl:51
 [4] (::Base.Pkg.Dir.##3#6{Array{Any,1},Base.Pkg.Entry.#add,Tuple{String}})() at .\pkg\dir.jl:33
 [5] cd(::Base.Pkg.Dir.##3#6{Array{Any,1},Base.Pkg.Entry.#add,Tuple{String}}, ::String) at .\file.jl:59
 [6] withenv(::Base.Pkg.Dir.##2#5{Array{Any,1},Base.Pkg.Entry.#add,Tuple{String},String}, ::Pair{String,String}, ::Vararg{Pair{String,String},N} where N) at .\env.jl:157
 [7] #cd#1(::Array{Any,1}, ::Function, ::Function, ::String, ::Vararg{String,N} where N) at .\pkg\dir.jl:32
 [8] add(::String) at .\pkg\pkg.jl:117
```

Do you know what the conflict might be?

**Comment by jmboehm** [comment]
Could it be that you have fixed DataFrames.jl?
https://docs.julialang.org/en/stable/manual/packages/#Updating-Packages-1

**Comment by lbittarello** [comment]
I don't think so... I have the latest release (v0.11.1) and I didn't modify or pin it.

**Comment by jmboehm** [comment]
Yes, I guess my dependencies are still on v0.10.1.

**Comment by jmboehm** [comment]
Should be fixed by c0de1336ee90899152f1a65beee449611a0495ab
